src/ontology/ro-edit.owl

```diff
@@ -4215,13 +4217,15 @@ SubObjectPropertyOf(obo:RO_0002468 obo:RO_0002465)
 
 # Object Property: obo:RO_0002469 (provides nutrients for)
 
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0002469 "A feeding relationship where one organism provides nutrition for another organism without being directly consumed. Examples include but are not limited to producing an exudate that is consumed by other organisms or transformation of nutrients into more labile forms.")
```

Comment:
Where inverse relations occur I recall RO is moving towards the pattern of just saying in defn. that "inverse of [inverse relation label]".